src/test/java/org/fundacionjala/coding/dennis/Twist37Test.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+
+/**
+ *
+ * @author DennisMont
+ *
+ */
+public class Twist37Test {
+
+    private final Integer[] numbersTest1Exp = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private final Integer[] numbersTest1 = new Integer[]{1, 2, 3, 4, 5, 6, 7, 8, 9};
+
+    private final Integer[] numbersTest2Exp = new Integer[]{12, 14, 13};
+    private final Integer[] numbersTest2 = new Integer[]{12, 13, 14};
+
+    private final Integer[] numbersTest3Exp = new Integer[]{2, 7, 4, 3, 9};
+    private final Integer[] numbersTest3 = new Integer[]{9, 2, 4, 7, 3};
```

Comment:
@AT-08/at-08 
These constants should be declared as local final variable inside test method.